### ISSUE TITLE
'struct.error: required argument is not an integer' when inviting a user bot to a group

### ISSUE THREAD

**Issue by kuba-orlik** [tulir/mautrix-telegram#443]
Inviting a user bot to a Telegram group does not trigger an invitation on Matrix. The mautrix-telegram logs show this message:

```
mautrix_1     | Traceback (most recent call last):
mautrix_1     |   File "/usr/lib/python3.8/site-packages/mautrix_telegram/portal/metadata.py", line 244, in create_matrix_room
mautrix_1     |     return await self._create_matrix_room(user, entity, invites)
mautrix_1     |   File "/usr/lib/python3.8/site-packages/mautrix_telegram/portal/metadata.py", line 259, in _create_matrix_room
mautrix_1     |     entity = await self.get_entity(user)
mautrix_1     |   File "/usr/lib/python3.8/site-packages/mautrix_telegram/portal/base.py", line 233, in get_entity
mautrix_1     |     return await user.client.get_entity(self.peer)
mautrix_1     |   File "/usr/lib/python3.8/site-packages/telethon/client/users.py", line 283, in get_entity
mautrix_1     |     chats = (await self(
mautrix_1     |   File "/usr/lib/python3.8/site-packages/telethon/client/users.py", line 53, in __call__
mautrix_1     |     future = self._sender.send(request, ordered=ordered)
mautrix_1     |   File "/usr/lib/python3.8/site-packages/telethon/network/mtprotosender.py", line 172, in send
mautrix_1     |     state = RequestState(request, self._loop)
mautrix_1     |   File "/usr/lib/python3.8/site-packages/telethon/network/requeststate.py", line 17, in __init__
mautrix_1     |     self.data = bytes(request)
mautrix_1     |   File "/usr/lib/python3.8/site-packages/telethon/tl/functions/messages.py", line 1141, in __bytes__
mautrix_1     |     b'\x15\xc4\xb5\x1c',struct.pack('<i', len(self.id)),b''.join(struct.pack('<i', x) for x in self.id),
mautrix_1     |   File "/usr/lib/python3.8/site-packages/telethon/tl/functions/messages.py", line 1141, in <genexpr>
mautrix_1     |     b'\x15\xc4\xb5\x1c',struct.pack('<i', len(self.id)),b''.join(struct.pack('<i', x) for x in self.id),
mautrix_1     | struct.error: required argument is not an integer
```

**Comment by kuba-orlik** [comment]
When trying to manually bridge a room via chat commands in Riot, I get:

![image](https://user-images.githubusercontent.com/2697916/77318537-21f21a00-6d05-11ea-9a31-93e8899d2d64.png)

**Comment by BjWe** [comment]
I am absolutely not into python,
but I found out (on my server) that 
there is no list with id's passed - instead a list of "telethon.tl.types.InputPeerChat".

The id is packed into id[n].chat_id.

I edited /telethon/tl/functions/messages.py and
added on top of
`class GetChatsRequest(TLRequest):`
`def __bytes__(self):`

```
  if self.id[0].__class__.__name__ == 'InputPeerChat':
            return b''.join((
                b'\x87\xa1j<',
                b'\x15\xc4\xb5\x1c',struct.pack('<i', len(self.id)),b''.join(struct.pack('<i', x.chat_id) for x in self.id),
            ))
```
That works for me now.
Not the best solution - but maybe it offers the developer a good hint
